pkg/virtual/syncer/controllers/apireconciler/syncer_apireconciler_reconcile.go

```diff
@@ -176,6 +152,60 @@ func gvrString(gvr schema.GroupVersionResource) string {
 	return fmt.Sprintf("%s.%s.%s", gvr.Resource, gvr.Version, group)
 }
 
+func (c *APIReconciler) getAllCompatibleResourceSchemas(syncTarget *workloadv1alpha1.SyncTarget) (map[schema.GroupResource]*apisv1alpha1.APIResourceSchema, map[schema.GroupResource]string, error) {
```

Comment:
What does `compatible` mean here? As I read it, this function accrues all schemas from all referenced exports that are already synced?

Reply:
this is the followup of this one https://github.com/kcp-dev/kcp/pull/1624 which check if synced resource is compatible with schema defined in exports.

Reply:
If you could define compatibility - perhaps in a comment for this method - that would help me wrap my head around this logic

Reply:
comments added. PTAL